src/agent/dashboard.rs

```diff
@@ -104,9 +102,7 @@ impl MetricsServer {
                 };
 
                 let last_local_update_string = if let Some(local_data) = price_data.local_data {
-                    if let Some(datetime) =
-                        NaiveDateTime::from_timestamp_opt(local_data.timestamp, 0)
-                    {
+                    if let Some(datetime) = DateTime::from_timestamp(local_data.timestamp, 0) {
```

Comment:
This was a suggested change due to deprecation of the previous method, it returns a DateTime<Utc> which has the same behaviour as the NaiveDateTime for formatting (because the inner NaiveDateTime is the same and Utc offset is 0).